phylogenetic/rules/prepare_sequences_E.smk

```diff
@@ -0,0 +1,94 @@
+"""
+This part of the workflow prepares sequences for constructing the phylogenetic tree.
+REQUIRED INPUTS:
+    metadata_url    = url to metadata.tsv.zst
+    sequences_url   = url to sequences.fasta.zst
+    reference   = path to reference sequence or genbank
+OUTPUTS:
+    prepared_sequences = results/aligned.fasta
+This part of the workflow usually includes the following steps:
+    - augur index
+    - augur filter
+    - augur align
+    - augur mask
+See Augur's usage docs for these commands for more details.
+"""
+
+ruleorder: nextclade3_cut_E > decompress
+ruleorder: filter_E > align
+
+rule generate_E_reference_files:
+    """
+    Generating reference files for the E gene
+    """
+    input:
+        reference = "config/reference_dengue_{serotype}.gb",
+    output:
+        fasta = "results/config/reference_dengue_{serotype}_E.fasta",
+        genbank = "results/config/reference_dengue_{serotype}_E.gb",
+    params:
+        gene = "E",
+    shell:
+        """
+        python3 bin/newreference.py \
+            --reference {input.reference} \
+            --output-fasta {output.fasta} \
+            --output-genbank {output.genbank} \
+            --gene {params.gene}
+        """
+
+rule nextclade3_cut_E:
```

Comment:
Suggest renaming this rule to drop the "nextclade3" bit and name it `align_E` or similar.

Reply:
Agree, fixed in https://github.com/nextstrain/dengue/pull/18/commits/27fa7c087adac0c6a02c5c8ba667fd6825cf846b